src/recommendations/ProductCard/BaseCard/index.jsx

```diff
@@ -20,7 +20,7 @@ const BaseCard = ({
   <div className="recommendation-card" key={`container-${uuid}`}>
     <Hyperlink
       target="_blank"
-      className="card-box"
+      className="card-box remove-inline-flex"
```

Comment:
can we use `d-inline`? and for my understanding, how did it fix the issue?

Reply:
Certainly, we can utilize it. However, the application of `display: inline-flex `disrupts the CSS.